CHANGELOG.md

```diff
@@ -1,5 +1,95 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.7
+
+<!-- generated comparing v5.15.6..master -->
+
+_Jan 30, 2024_
+
+A big thanks to the 21 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.15.7`
+
+- &#8203;<!-- 52 -->[Alert] Add action, icon, and iconMapping tests (#40682) @DiegoAndai
```

Comment:
This should go under core if it's only about adding tests, we should have also added the [test] tag.